src/server.js

```diff
@@ -1 +1,81 @@
-// Write your code here
\ No newline at end of file
+const express = require('express');
+const fs = require('fs');
+const path = require('path');
+
+const app = express();
+const port = 5000;
+
+app.get('/movies', (req, res) => {
+  fs.readFile(path.join(__dirname, '../data.json'), (err, content) => {
+    if (err) {
+      res.status(500).json({
+        statusCode: 500,
+        errorMessage: err,
+        errorDescription: 'Error read file data.json',
+      });
+    } else {
+      films = JSON.parse(content);
```

Comment:
where is variable declaration? Code will not work with `"use strict";`. Check other places too

Reply:
> where is variable declaration? Code will not work with `"use strict";`. Check other places too

refactored